src/oic/oauth2/__init__.py

```diff
@@ -425,6 +425,8 @@ def construct_AuthorizationRequest(
         else:
             request_args = {}
 
+        request_args.update(kwargs)
```

Comment:
My problem is that if I don't add the request args manually from kwargs these arguments like client_id, redirect_uri passed to the oic.oic. construct_AuthenticationRequest method are never added to the authentication request. So, if I'm not doing anything else wrong, there is a problem when picking the args passed to this method because if I run my app with the authenticationRequest method detailed below without my changes the redirect_uri arg in the authentication request made to the OP is equal to "h". When I use the python debug console to print session["client"].registration_response["redirect_uris"][0] its value is 'https://localhost:8080/authenticateDigitelTS' and therefore it's correct. 

```
def authenticationRequest(self, session):
        """
        Sends a request to the OP to authenticate the user that made this request.

        :param session: a CherryPy Session with the session of the user that made the request.

        :return a RegistrationResponse instance with the response.
        """

        session["state"] = rndstr()
        session["nonce"] = rndstr()
        args = {
            "client_id": session["client"].client_id,
            "response_type": session["client"].response_type,
            "scope": session["client"].scope,
            "nonce": session["nonce"],
            "redirect_uri": session["client"].registration_response["redirect_uris"][0],
            "state": session["state"]
        }

        auth_request = session["client"].construct_AuthorizationRequest(**args)
        # Send the request to the authorization endpoint.
        loginPseudoURL = auth_request.request(session["client"].authorization_endpoint)

        return loginPseudoURL
```

I don't know if the problem is because of these arguments never being added to request_args in oic.oauth2.construct_AuthorizationRequest or there is an error in any other place because I don't know all the implementation details of the library that well. Maybe you can point me in the right direction. Thanks for your help, have a good day.

PD: I will check google documentation conventions for future PRs